packages/wonder-blocks-core/src/hooks/use-latest-ref.ts

```diff
@@ -10,7 +10,7 @@ import * as React from "react";
  * be passed to useEffect without causing unnecessary re-renders. See the
  * Storybook docs for a usage example.
  */
-export function useLatestRef<T>(value: T): React.RefObject<T> {
+export function useLatestRef<T>(value: T): {readonly current: T} {
```

Comment:
**suggestion:** Could do `Readonly<React.MutableRefObject<T>>`? :shrug: If you change it, don't forget to update the docs.